cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,93 @@
+/* eslint-disable cypress/unsafe-to-chain-command */
+/* eslint-disable arrow-parens */
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  it('should have pagination', () => {
+    // pagination
+    cy.get('.-pageInfo').should('contain', 'Page');
+    cy.get('.-pageJump > input').type('Number');
+    cy.get('.-totalPages').should('contain', '1');
+    cy.get('.-btn').should('contain', 'Previous');
+    cy.get('.-btn').should('contain', 'Next');
+  });
+
+  it('should have rows count selection', () => {
+    // Rows count selection
+    cy.get('select').should('contain', '5 rows');
+    cy.get('select').should('contain', '10 rows');
+    cy.get('select').should('contain', '20 rows');
+    cy.get('select').should('contain', '25 rows');
+    cy.get('select').should('contain', '50 rows');
+    cy.get('select').should('contain', '100 rows');
+  });
+
+  it('should allow to add a new worker', () => {
+    cy.get('#addNewRecordButton').should('contain', 'Add').click();
```

Comment:
It is not necessary to check this button, because you will create a new user after clicking it, so if there be some errors during clicking your user will not be added

Reply:
I've got it. Done.